text/metrics/0146-metrics-prototype-scenarios.md

```diff
@@ -0,0 +1,220 @@
+# Scenarios for Metrics API/SDK Prototyping
+
+With the stable release of the tracing specification, the OpenTelemetry
+community is willing to spend more energy on metrics API/SDK. The goal is to get
+the metrics API/SDK specification to
+[`Experimental`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#experimental)
+state by end of 5/2021, and make it
+[`Stable`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable)
+before end of 2021:
+
+* By end of 5/2021, we should have a good confidence that we can recommend
+  language client owners to work on metrics preview release. This means starting
+  from 7/1/2021 the specification should not have major surprise or big changes
+  which would thrust the language client maintainers.
+
+* By end of 9/2021, we should mark the metrics API/SDK specification as
+  [`Feature-freeze`](https://github.com/open-telemetry/opentelemetry-specification/blob/1afab39e5658f807315abf2f3256809293bfd421/specification/document-status.md#feature-freeze),
+  and focusing on bug fixing or editorial changes.
+
+* By end of 2021, we want to have a stable release of metrics API/SDK
+  specification, with multiple language SIGs providing RC (release candidate) or
+  [stable](https://github.com/open-telemetry/opentelemetry-specification/blob/9047c91412d3d4b7f28b0f7346d8c5034b509849/specification/versioning-and-stability.md#stable)
+  clients.
+
+In this document, we will focus on two scenarios that we use for prototyping
+metrics API/SDK. The goal is to have two scenarios which clearly capture the
+major requirements, so we can work with language client SIGs to prototype,
+gather the learnings, determine the scopes and stages. Later the scenarios can
+be used as examples and test cases for all the language clients.
+
+Here are the languages we've agreed to use during the prototyping:
+
+* C#
+* Java
+* Python
+
+Instead of boiling the ocean, we will need to divide the work into multiple
+stages:
+
+1. Do the end-to-end prototype to get the overall understanding of the problem
+   domain. We should also clarify the scope and be able to articulate it
+   precisely during this stage, here are some examples:
+
+    * Why do we want to introduce brand new metrics APIs versus taking a well
```

Comment:
This point (and the next point) beg the question:

Should we make ONE of the use cases be "hide OTEL behind another library to help users take advantage of OTEL telemetry-unfiication concepts, like Baggage + Resource".   This could be Micrometer, OpenCensus, whatever.